### DISCUSSION TITLE
Latest version ? What's needed for Ventura?

### DISCUSSION THREAD

**Discussion by ccgr33n** [ExistentialAudio/BlackHole#690] [Q&A]
I see a release for v0.4.0. 
I am interested in updating to Ventura and want to be sure I have the latest version (per some discussions I found).
When I asked before how to find out what version I am on I was told:
/Library/Audio/Plug-Ins/HAL/BlackHole2ch.driver/Contents/Info.plist
And looking at "Bundle version string (short)”.
I got to info.plist - opened in TextEdit and found this, that would be closed to a version:
<string>0.2.9</string>

 I want to update to Ventura, and see a few comments that it appears to be working, but may need to update the driver. 

I went to the download - and signed in to get the email link. I downloaded it; however, the plist file still shows the old release, not the update. I did a search of parts of the driver name- and could not find it anywhere.  So I am clearly missing something (though came a long way to get this far).
Please advise.

Much appreciated.

Stay safe. Remain healthy. Enjoy life.

**Comment by MartGHub** [comment]
As far as I know I'm running the latest version (I downloaded it again last week).  Since upgrading MacOS to Ventura I'm getting very high levels of latency when I try to play a YT video (e.g. a backing track or song I am learning) and use Helix Native (a guitar/bass plug-in running in GarageBand) at the same time.  I'm getting the same latency issue with other sound sources also, such as Ireal Pro (which isn't streaming information from the internet).  When I switch my audio routing to just use my audio interface and take Blackhole out of the signal path the latency issues disappear.  

Thought I'd let you know as it's quite a frustrating issue.  Hopefully it can be addressed in the next update?

**Comment by devinroth** [comment]
Check the version. BlackHole.driver/contents/info.plist

And a screenshot of your aggregate.

Devin Roth
